app/css/_sidebar.scss

```diff
@@ -0,0 +1,68 @@
+.sidebar-right-aside {
+  min-width: $min-width;
+  float: left;
```

Comment:
Probably this is only needed starting from tablets...
Then width is also not needed.